resources/serverless/values.yaml

```diff
@@ -217,7 +217,7 @@ containers:
       functionJaegerServiceEndpoint:
         value: "http://tracing-jaeger-collector.kyma-system.svc.cluster.local:14268/api/traces"
```

Comment:
@chrkl would this legacy endpoint continue to work ?

Reply:
For now it still works, but would not recommend to use it since the otel collector adds some workload related attributes to the spans (pod name, namespace, etc.). On long term, Jaeger will be removed from Kyma and be replaced by a tutorial.

Is it still in use? Our otel collector does not provide a Jaeger endpoint that can be used as a replacement.

Reply:
If it's going to be deleted, we need to reload all functions @kwiatekus

Reply:
For all new/Updated functions the new collector endpoint will be  used. However there might be still functions that were injected with the old value of the url and will continue to use it even after kyma upgrade.. unless we enforce reloading of those envs somehow

Reply:
We will not remove jaeger now, so the old endpoint will continue to work for a while. For an SKR scenario it is a new feature to have tracing at all. I think it is fine to not do any additional actions around that.